tests/conftest.py

```diff
@@ -16,6 +16,8 @@
 PROJECT_ID = os.environ.get('GOOGLE_BIGQUERY_PROJECT_ID', 'ibis-gbq')
 DATASET_ID = 'testing'
 
+bq = ibis_bigquery.Backend()
```

Comment:
@datapythonista I notice that in the OmniSciDB repo, there isn't a `Backend` class and `connect` is a top-level function. What approach should we be following in ibis-bigquery?

Reply:
I'm not keeping the Omnisci backend updated with the refactoring in Ibis. I think it'll be easy to make the changes at once, after the backend public API is finished (after Ibis 2.0 is released). The bigquery backend should be updated to the last changes (since the tests don't run in PRs maybe I missed something). But the `Backend` class is more close to what we'll have in Ibis 2.0 than what Omnisci has.

In the future, I think I'll merge `Backend` with `Client`, but what's the best API will be more obvious as I keep simplifying things.